orion/run_admin_local.bat

```diff
@@ -0,0 +1,17 @@
+:: This script builds the website locally AND runs the orion server locally
+:: So you may interact with the website and api server end to end.
+
+:: First step is to make sure mysql is installed on your computer and running
+:: net start MySQL
+:: net stop MySQL
+:: mysql db --user=root --password
+
+:: Second step is to build the website locally
+cd sites\admin
+call npm run build-local
```

Comment:
You need a "call" in front because of an issue between batch and npm
https://github.com/npm/npm/issues/2938